common/app/experiments/Experiments.scala

```diff
@@ -54,3 +55,12 @@ object OphanNext
       sellByDate = LocalDate.of(2024, 4, 1),
       participationGroup = Perc1B,
     )
+
+object DCRTagFronts
+    extends Experiment(
+      name = "dcr-tag-fronts",
+      description = "Render tag fronts with DCR",
+      owners = Seq(Owner.withGithub("dotcom.platform@theguardian.com")),
+      sellByDate = LocalDate.of(2024, 8, 5),
```

Comment:
That seems too far into the future for a 1% test… I would suggest a month at most before requiring a review of increasing the percentage.

Reply:
How ambitious! I like it.